packages/pretty-format/src/__tests__/react.test.js

```diff
@@ -49,13 +53,20 @@ test('supports a single element with no props', () => {
   );
 });
 
-test('supports a single element with number children', () => {
+test('supports a single element with nonzero number child', () => {
```

Comment:
Yes, thank you. And I moved the new test for empty string to make the pairs match up.